src/pages/deriv-go/_what-is-deriv-go.tsx

```diff
@@ -91,7 +90,7 @@ const WhatIsDerivGo = () => {
                         <StyledTitle as="h3" type="heading-3" align="center">
                             {item.title}
                         </StyledTitle>
-                        <StyledSubTitle align="center" weight="100">
+                        <StyledSubTitle align="center" weight="400" as="span">
```

Comment:
we can provide `type` here as well and remove font-size properties from component at all